src/core/sm/networkmanager/itf/networkmanager.hpp

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CORE_SM_NETWORKMANAGER_ITF_NETWORKMANAGER_HPP_
+#define AOS_CORE_SM_NETWORKMANAGER_ITF_NETWORKMANAGER_HPP_
+
+#include <core/common/tools/memory.hpp>
+#include <core/common/types/network.hpp>
+#include <core/sm/config.hpp>
+
+#include "interfacefactory.hpp"
+#include "interfacemanager.hpp"
+#include "namespacemanager.hpp"
+#include "storage.hpp"
+#include "trafficmonitor.hpp"
+
+namespace aos::sm::networkmanager {
+
+/** @addtogroup sm Service Manager
+ *  @{
+ */
+
+/**
+ * Max number of network manager aliases.
+ */
+static constexpr auto cMaxNumAliases = AOS_CONFIG_NETWORKMANAGER_MAX_NUM_ALIASES;
+
+/**
+ * Max number of hosts.
+ */
+static constexpr auto cResolvConfLineLen = AOS_CONFIG_NETWORKMANAGER_RESOLV_CONF_LINE_LEN;
```

Comment:
Looks like not used in this interface. Consider to move to private section where it is used.

Reply:
Done